pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: shelf_gzip
 description: Shelf middleware to GZIP encoding responses, with compression level and compression scope by content-type.
-version: 4.0.1
+version: 4.0.2
```

Comment:
nit: 4.1.0, since this includes a change to the API.

Reply:
ok